internal/graph/urn.go

```diff
@@ -17,18 +22,37 @@ func MakeURN(s, bucket string) (string, error) {
 	s2c := getLastTwo(sr) // split the string and take last two segments
 
 	if strings.Contains(s2c, ".rdf") {
-		g = fmt.Sprintf("urn:%s:%s", bucket, strings.TrimSuffix(s2c, ".rdf"))
+		g = fmt.Sprintf("urn:%s:%s:%s", bucket, gcfg.Orgname, strings.TrimSuffix(s2c, ".rdf"))
```

Comment:
bucket should not be in the urn.  "gleaner.io" would be string.

We might need a program/handle option in the implementation_net config, if we want people to change this.
Goal is that I can change the resource location underneath the system.

Reply:
The ADR was wrong too..  will update decision.   I am fine with this.  I don't think the path needs to be encoded in the URN anymore.  Especially with the release graphs being the main way people will get to the resources.